### PR TITLE
update: card content

### PR DESCRIPTION
**Pull request by terran6**
Adding card to root pages and fixing pagination SCSS file.

### CODE REVIEW

File: src/components/Cards/card-data/develop-cards.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    darkSvg: '/img/icons/dark/localterra.svg',
+    header: 'LocalTerra',
+    summary: 'Run a local Terra blockchain in Docker.'
+  },
+  {
+    href: 'https://faucet.terra.money/',
+    lightSvg: '/img/icons/Faucet.svg',
+    darkSvg: '/img/icons/dark/Faucet.svg',
+    header: 'Terra Testnet Faucet',
+    summary: 'Receive test Luna to carry out transactions on Testnet.'
```
Comment (evanorti) [inline src/components/Cards/card-data/develop-cards.js:42]: Change to "the testnet"?  
"Testnet" is not a proper noun anywhere else and not capitalized unless it's at the beginning of a sentence.

File: src/components/Cards/card-data/full-node-cards.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+const fullNodeCards = [
+  {
+    href: '/full-node/run-a-full-terra-node',
+    lightSvg: '/img/icons/node.svg',
+    darkSvg: '/img/icons/dark/node.svg',
+    header: 'Run a full node',
+    summary: 'Walk through the configuration and management of a full node on Terra.'
```
Comment (evanorti) [inline src/components/Cards/card-data/full-node-cards.js:7]: The phrasing used in other parts of the docs is "a full Terra node." 

So I would change this to "Walk through the configuration and management of a full Terra node."

or simply "Configure and manage a full Terra node."

LMK your thoughts on this.

File: src/components/Cards/card-data/full-node-cards.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  {
+    href: '/full-node/run-a-full-terra-node/join-a-network.html',
+    lightSvg: '/img/icons/ecosystem02.svg',
+    darkSvg: '/img/icons/dark/ecosystem02.svg',
+    header: 'Join a network',
+    summary: 'Set up a full node on various Terra networks.'
```
Comment (evanorti) [inline src/components/Cards/card-data/full-node-cards.js:14]: I think this card should be more specific:

"Join the mainnet, testnet, or make a local private network. "

File: src/components/Cards/card-data/terra-station-cards.js [hunk 4]
```diff
@@ -1,66 +1,59 @@
 const terraStationCards = [
   {
-    href: './download/',
+    href: '/learn/terra-station/download/',
     lightSvg: '/img/icons/station/station.svg',
     darkSvg: '/img/icons/dark/station.svg',
     header: 'Download Station',
     summary: 'Download Terra Station, the official wallet for Terra.'
```
Comment (evanorti) [inline src/components/Cards/card-data/terra-station-cards.js:7]: I think grammatically "the official wallet for Terra" 
should be "the official wallet of Terra."

Or maybe:
"Download Terra Station, Terra's official wallet."

File: src/components/Cards/card-data/terra-station-cards.js [hunk 5]
```diff
@@ -1,66 +1,59 @@
...
     header: 'Create a wallet',
     summary: 'Create and manage a wallet.'
   },
   {
-    href: './networks.html',
+    href: 'https://station.terra.money/',
+    lightSvg: '/img/icons/Terrad.svg',
+    darkSvg: '/img/icons/dark/Terrad.svg',
+    header: 'Station WebApp',
+    summary: 'Utilize the Terra Station web application for a more complete experience.'
```
Comment (evanorti) [inline src/components/Cards/card-data/terra-station-cards.js:21]: I think we want to push people to download Station desktop more than use the WebApp? 

Unsure.

File: src/components/Cards/card-data/terra-station-cards.js [hunk 6]
```diff
@@ -1,66 +1,59 @@
...
+  },
+  {
+    href: '/learn/terra-station/networks.html',
     lightSvg: '/img/icons/swap-1.svg',
     darkSvg: '/img/icons/dark/swap-1.svg',
     header: 'Switch networks',
     summary: 'Switch between networks on Terra.'
```
Comment (evanorti) [inline src/components/Cards/card-data/terra-station-cards.js:28]: "Switch between the mainnet, testnet, or LocalTerra."

File: src/components/Cards/card-data/terrain-cards.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    darkSvg: '/img/icons/dark/smart-contract.svg',
+    header: 'Contract migration with Terrain',
+    summary: 'Learn how to migrate CosmWasm contracts on Terra.'
+  },
+  {
+    href: '/develop/terrain/using-terrain-testnet.html',
+    lightSvg: '/img/icons/testnet.svg',
+    darkSvg: '/img/icons/dark/testnet.svg',
+    header: 'Testnet development with Terrain',
+    summary: 'Learn how to interact with Testnet using Terrain.'
```
Comment (evanorti) [inline src/components/Cards/card-data/terrain-cards.js:21]: Change to "the testnet."

File: src/components/Cards/card-data/terra-station-cards.js (hunk 5 above)
Comment (terran6) [reply src/components/Cards/card-data/terra-station-cards.js:21]: @evanorti, think we should get rid of that card altogether?

Comment (evanorti) [reply src/components/Cards/card-data/terra-station-cards.js:21]: Yes. I think so

**Comment by evanorti** [comment]
LGTM

**Pull request merged**